### DISCUSSION TITLE
Welcome to ViPERFX_RE Discussions!

### DISCUSSION THREAD

**Discussion by iscle** [AndroidAudioMods/ViPERFX_RE#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Web-Cooler** [comment]
啥时候能用上啊 在等大佬们做出来
